hhvm.rb

```diff
@@ -241,6 +241,14 @@ def install
       -DZLIB_LIBRARY=/usr/lib/libz.dylib
     ]
 
+    # Compile with PGSQL
+    cmake_args += %W[
+      -DENABLE_EXTENSION_PGSQL=ON
+      -DPGSQL_INCLUDE_DIR=/usr/local/opt/postgres/include
+      -DPGSQL_LIBRARY=/usr/local/opt/postgres/lib/libpq.dylib
```

Comment:
Where do these files come from? Shouldn't we be taking a dependency on brew postgres and using that, instead of these paths, which I don't think exist on a stock OS X install? What happens if someone doesn't have these installed?